ncm-dpmlfc/src/test/perl/configure.t

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/perl
 use strict;
 use warnings;
+
+BEGIN {
+    *CORE::GLOBAL::sleep = sub {};
+}
+
```

Comment:
There's nothing in your commit message or comments about these changes, what are they for?